app/api/unified-aegenesis-api.legacy.yml

```
@@ -0,0 +1,49 @@
+# Legacy unified AeGenesis API specification (archived)
+# This file was archived after fragment-based assembly was introduced.
+# Authoritative spec is now generated at: build/openapi/unified-aegenesis-api.generated.yml
+
+openapi: 3.1.0
+info:
+  title: AeGenesis Coinscience AI Ecosystem API (Legacy)
+  version: 1.0.0-genesis-coinscience
+  description: Archived legacy monolithic specification. Do not modify.
+  x-archived-at: 2025-09-05T00:00:00Z
+# --- Original Content Below ---
+openapi: 3.1.0
```

YAMLlint (1.37.1)

[error] duplication of key "openapi" in mapping

(key-duplicates)